Sources/BraveWallet/Crypto/Stores/PortfolioStore.swift

```diff
@@ -502,6 +510,26 @@ public class PortfolioStore: ObservableObject, WalletObserverStore {
           formattedPrice: currencyFormatter.string(from: NSNumber(value: value)) ?? "0.00"
         )
       }
+      
+      if let oldestHistoricalValue = historicalBalances.first {
+        let priceDifference = currentBalance - oldestHistoricalValue.price
+        let percentageChange = priceDifference / oldestHistoricalValue.price * 100
+        let isBalanceUp = priceDifference > 0
+        balanceDifference = .init(
+          priceDifference: String(
+            format: "%@%@",
+            isBalanceUp ? "+" : "", // include plus if balance increased
```

Comment:
there is no "-" for down?

Reply:
The `-` gets added automatically for negative numbers 🙂
![down](https://github.com/brave/brave-ios/assets/5314553/220d3e78-8ec8-4a3b-83fa-f4c01b04ef49)